packages/reakit/src/Popover/PopoverState.ts

```diff
@@ -100,7 +100,11 @@ export type PopoverInitialState = DialogInitialState &
      */
     unstable_inner?: boolean;
     /**
-     * Offset between the reference and the popover.
+     * Offset between the reference and the popover: [main axis, alt axis]. Should not be combined with `gutter`.
+     */
+    unstable_offset?: [number | string, number | string];
```

Comment:
Interestingly, the tuple type here causes an issue at my callsite, where typescript assumes

```ts
unstable_offset: offset = ['50%p', '-75%p'],
```

is actually of array type `(number | string)[]`, and not a tuple type. Have you come across this before? Any suggestions for resolving? I'm able to get typescript to understand the correct tuple type by manually re-casting my callsite type, but that seems suboptimal.

I could update the type here to just be an array type and log a warning in case it's not exactly length two, but that feels less typesafe than debugging the issue with the tuple type.

Reply:
If that's the same issue I saw before, the best solution I can think of is to cast it using `as const`. I'd also try declaring the type with `ReadonlyArray` or something.